app/components/speaker-list.js

```diff
@@ -0,0 +1,6 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
```

Comment:
Use destructuring here to pass `eslint`.

Ref: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment 

See other components to better understand how it's done.